src/index.tsx

```diff
@@ -69,6 +69,34 @@ const renderApp = (state) => {
         });
     }
 
+    const updateNewOrgTitle = (newOrgTitle) => {
+        renderApp({
+            ...state,
+            newOrgTitle
```

Comment:
Here u add a property called newOrgTitle to the state; but it needs to be inside the organizations array